dGame/dInventory/ItemSet.cpp

```diff
@@ -27,14 +27,16 @@ ItemSet::ItemSet(const uint32_t id, InventoryComponent* inventoryComponent) {
 		return;
 	}
 
-	for (auto i = 0; i < 5; ++i) {
-		if (result.fieldIsNull(i)) {
+	const std::vector<std::string_view> rowNames = { "skillSetWith2", "skillSetWith3", "skillSetWith4", "skillSetWith5", "skillSetWith6" };
```

Comment:
nit: could this be a constexpr std::array?

Reply:
Changed it to be one.

Reply:
I would appreciate not editing my PRs unless I have asked someone to, or worked together with someone on it.  I did not think those changes were necessary and would have left them alone personally as they would not change anything.

Reply:
I was not aware of that. Will do moving forward.